GitUI/CommandsDialogs/SettingsDialog/SettingsTreeViewUserControl.cs

```diff
@@ -124,7 +123,7 @@ private void textBoxFind_TextChanged(object sender, EventArgs e)
         {
             _nodesFoundByTextBox = new List<TreeNode>();
 
-            if (string.IsNullOrEmpty(textBoxFind.Text) || textBoxFind.Text == FindPrompt)
+            if (string.IsNullOrEmpty(textBoxFind.Text) || textBoxFind.Text == TranslatedStrings.SettingsTypeToFind)
```

Comment:
I suppose we could use [`PlaceHolderText`](https://docs.microsoft.com/dotnet/api/system.windows.forms.textbox.placeholdertext) instead of setting the text to `TranslatedStrings.SettingsTypeToFind`.

Reply:
Updated. I made the change to `PlaceHolderText` (that simplify remove a whole bunch of code) but unfortunately doesn't prevent to use the `TranslatedStrings` because this field is not supported by the translation.